docs/03-compute-resources.md

```diff
@@ -98,23 +98,20 @@ Create three compute instances which will host the Kubernetes **control plane**.
 - Execute "yum update -y" once the server is up and running. This command is injected into the cloudinit, so all instances are up to date since the very beginning.
 
 ```
-# for node in master00 master01 master02; do
```

Comment:
I found it much more easier to understand by someone who does not know kcli instead of a plan. Do you think an introduction to what's a plan should be created?

Reply:
no it s fine this way